Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,7 +10,56 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A."; 
+            }
+
+            var HighestBalanceEver = decimal.MinValue; 
+            var PersonWithHighestBalance = new StringBuilder() ; 
+
+            for (int i = 0; i <= peopleAndBalances.Length - 1; i++)
+			{
+                var currentPersonData = peopleAndBalances[i].Split(',') ; 
+                var highestAmountOfPerson = decimal.Parse(currentPersonData[1 ..].Max());
+
+                if (highestAmountOfPerson > HighestBalanceEver)
+                {
+                    HighestBalanceEver = highestAmountOfPerson;
+                    var currentOne = PersonWithHighestBalance.ToString();
+                    
+                    if (string.IsNullOrEmpty(currentOne))
+                    {
+                        PersonWithHighestBalance.Append(currentPersonData[0]);
+                    }
+                    else
+                    {
+                        PersonWithHighestBalance.Replace(currentOne, currentPersonData[0]);
+                    }
+
+
+                } else if (highestAmountOfPerson == HighestBalanceEver)
+                {
+                    if (i == peopleAndBalances.Length -1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4929752)